src/Sarif.Viewer.VisualStudio.UnitTests/Models/AnalysisStepNodeTests.cs

```diff
@@ -11,6 +14,8 @@ namespace Microsoft.Sarif.Viewer.Models
 {
     public class AnalysisStepNodeTests
     {
+        private static readonly Random random = new Random();
```

Comment:
>random [](http://example.com/codeflow?start=39&length=6)

I remember Michael mention add s_ to static vars

Reply:
updated